### ISSUE TITLE
Subroutines redefined from utf8_heavy in compiled code

### ISSUE THREAD

**Issue by atoomic** [rurban/perl-compiler#369]
Notice this problem with several cPanel binaries during destruction
```
Subroutine DESTROY redefined at /usr/local/cpanel/3rdparty/perl/522/lib64/perl5/5.22.1/utf8_heavy.pl line 10.
Subroutine croak redefined at /usr/local/cpanel/3rdparty/perl/522/lib64/perl5/5.22.1/utf8_heavy.pl line 14.
Subroutine _loose_name redefined at /usr/local/cpanel/3rdparty/perl/522/lib64/perl5/5.22.1/utf8_heavy.pl line 16
```

This shows that utf8 heavy is compiled in but not marked correctly.
Unfortunately I'm not able to provide a short one liner to reproduce the error, and the short script that I have here requires cPanel codebase

```
BEGIN { require "bin/admin/Cpanel/notify_call.pl"; }
print qq{$_\n} foreach sort keys %INC;
eval q{ require Email::MIME::Encode };
print qq{ok\n};
```

```
> perlcc test.pl && ./test
....
....
utf8.pm
warnings.pm
warnings/register.pm
Subroutine DESTROY redefined at /usr/local/cpanel/3rdparty/perl/522/lib64/perl5/5.22.1/utf8_heavy.pl line 10.
Subroutine croak redefined at /usr/local/cpanel/3rdparty/perl/522/lib64/perl5/5.22.1/utf8_heavy.pl line 14.
Subroutine _loose_name redefined at /usr/local/cpanel/3rdparty/perl/522/lib64/perl5/5.22.1/utf8_heavy.pl line 16.
ok
```

**Comment by atoomic** [comment]
@rurban any concerns with the suggested patch for this case  ?

**Comment by rurban** [comment]
Fixed with 96460e21829cfc8fcebb1ac3fa56f99cbfacbd0d
